WebIDL/ecmascript-binding/interface-prototype-object.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Interface prototype objects</title>
+<link rel="help" href="https://heycam.github.io/webidl/#interface-prototype-object">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+// A specification issue was raised for this behavior.
+// https://www.w3.org/Bugs/Public/show_bug.cgi?id=28244
+test(function() {
+    assert_class_string(Object.getPrototypeOf(Document.prototype),
+                        'DocumentPrototype');
```

Comment:
If we're testing the specced behavior, we should also be testing:

```js
assert_true(document.hasOwnProperty(Symbol.toStringTag));
assert_equals(document[Symbol.toStringTag] === "Document");

assert_true(Document.prototype.hasOwnProperty(Symbol.toStringTag));
assert_equals(Document.prototype[Symbol.toStringTag] === "DocumentPrototype");
```

Not even Firefox passes these tests, but I think it's more honest to have the tests document the fact that nobody conforms to the current spec.